mne/encoding/feature.py

```diff
@@ -0,0 +1,269 @@
+"""Classes for transforming, preprocessing, and fitting encoding models."""
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#          Jona Sassenhagen <jona.sassenhagen@gmail.de>
+#          Jean-Remi King <jeanremi.king@gmail.com>
+#
+# License: BSD (3-clause)
+import numpy as np
+from scipy.sparse import spmatrix, csr_matrix
+from ..utils import warn
+
+
+class FeatureDelayer(object):
```

Comment:
True. We Can just mention the Hankel in the doc string then . As long as we
maximize the name consistency between mne and pyriemann I m fine.

Le 21 août 2016 23:56, "alexandre barachant" <notifications@github.com> a
écrit :

> In mne/encoding/feature.py
> <https://github.com/mne-tools/mne-python/pull/3310#discussion_r75604193>:
>
> > @@ -0,0 +1,269 @@
> > +"""Classes for transforming, preprocessing, and fitting encoding models."""
> > +# Authors: Chris Holdgraf <choldgraf@gmail.com>
> > +#          Jona Sassenhagen <jona.sassenhagen@gmail.de>
> > +#          Jean-Remi King <jeanremi.king@gmail.com>
> > +#
> > +# License: BSD (3-clause)
> > +import numpy as np
> > +from scipy.sparse import spmatrix, csr_matrix
> > +from ..utils import warn
> > +
> > +
> > +class FeatureDelayer(object):
>
> Hankel would match, but does not speak much to most of us.
> something with 'time lag' might be more explicit.
>
> FYI, I called it HankelCovariances in pyRiemann when it include the
> covariance estimation, and i was about to push a new transformer with the
> ugly name of TimeDelayer :(
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3310/files/e9c40309c15a581b0901f5b0784aacca5d2e0e6c#r75604193>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEp7DFggawWmScO4jYP0f4rwEIhLCxIYks5qiMlpgaJpZM4I28Ut>
> .
>